boards/ESP32.md

```diff
@@ -146,6 +147,14 @@ esp-idf/components/esptool_py/esptool/esptool.py    \
         0x10000 espruino_esp32.bin
 ```
 
+If the `bootloader.bin` and `partitions_espruino.bin` files are not
+included from the same source you acquired the `espruino_esp32.bin` build,
+they should be available in the corresponding `espruino_1v92.*_esp32.tgz`
```

Comment:
Oops... should be `espruino_*_esp32.tgz`.  Anyway, it sounds like this addition will be imminently obsolete by the time it matters.  It could be removed then.